### PR TITLE
Architecture decision - dont inquire after notifications

### PR DESCRIPTION
**Pull request by heathd**
in order to better document significant architectural decisions in the
payments platform, let's use Architectural Decision Records[1]

These have been used on Registers[2] and on the GOV.UK Publishing API[3]
and have proven helpful in communicating software design decisions
within the team.

This ADR documents our decision to not perform `inquiry` calls to
Worldpay after receiving notifications.

[1] http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions
[2] https://github.com/openregister/presentation/pull/71
[3] https://github.com/alphagov/publishing-api/pull/84

### CODE REVIEW

**Comment by heathd** [comment]
rebuild

File: doc/arch/adr-001-dont-do-inquiry-after-notification.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      <orderStatus orderCode="01105e9b-f0e6-446c-a093-a2ef48f5fd39">
+         <payment>
+            <paymentMethod>VISA-SSL</paymentMethod>
+            <paymentMethodDetail>
+               <card type="creditcard" />
+            </paymentMethodDetail>
+            <amount value="500" currencyCode="GBP" exponent="2" debitCreditIndicator="credit" />
+            <lastEvent>REFUNDED</lastEvent>
+            <refundReference>refund-1-01105e9b-f0e6-446c-a093-a2ef48f5fd39</refundReference>
+            <reference>refund-1-01105e9b-f0e6-446c-a093-a2ef48f5fd39</reference>
```
Comment (rahul-vinayak) [inline doc/arch/adr-001-dont-do-inquiry-after-notification.md]: Does it include both <refundReference> and <reference> elements?

**Comment by rahul-vinayak** [comment]
👍

File: doc/arch/adr-001-dont-do-inquiry-after-notification.md (hunk 1 above)
Comment (heathd) [reply doc/arch/adr-001-dont-do-inquiry-after-notification.md]: yes it does come with both

**Pull request merged**